go-controller/pkg/node/node.go

```diff
@@ -495,6 +498,15 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	}
 
 	if config.HybridOverlay.Enabled {
+		var libovsdbNBClient libovsdbclient.Client
+		var err error
+
+		stopChan := make(chan struct{})
+		defer close(stopChan)
+
+		if libovsdbNBClient, err = libovsdb.NewNBClient(stopChan); err != nil {
```

Comment:
in the previous commit didnt you make a libovsdb client for the node, why would you need to make another one here? I think you can jsut pass into node.Start

Reply:
+1 Sorry, mess up between commits